tgui/packages/tgui/interfaces/CheckAntags.js

```diff
@@ -0,0 +1,331 @@
+/* eslint operator-linebreak: 0 */
+// https://github.com/jsx-eslint/eslint-plugin-react/issues/454
+
+import { useBackend } from '../backend';
+import { Box, Button, Divider, Grid, Input, LabeledList, NumberInput, Section } from '../components';
+import { Window } from '../layouts';
+
+export const CheckAntags = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    mode,
+    replacementmode,
+    time,
+    shuttlecalled,
+    shuttletransit,
+    shuttletime,
+    continue1,
+    continue2,
+    midround_time_limit,
+    midround_living_limit,
+    end_on_death_limits,
+    connected_players,
+    lobby_players,
+    observers,
+    observers_connected,
+    living_players,
+    living_players_connected,
+    living_players_antagonist,
+    brains,
+    other_players,
+    living_skipped,
+    drones,
+    antags,
+    antag_types,
+    priority_sections,
+    sections,
+  } = data;
+  return (
+    <Window
+      width={750}
+      height={710}
+      theme={'ntos'}>
+      {act('loadsuccess')}
+      <Window.Content scrollable>
+        <Grid height={15}>
+          <Grid.Column>
+            <Section title="Round Status" fill>
+              {replacementmode ? 'Former Game Mode: ' : 'Current Game Mode: '} <Box inline bold>{mode}</Box>
+              {replacementmode ? 'Replacement Game Mode: ' : ''} <Box inline bold>{replacementmode ? replacementmode : ''}</Box>
+              <br />
+              <br />
+              Round Duration:
+              <br />
+              <Box inline bold>{time}</Box>
+              <br />
+              <br />
+              <Box bold>Emergency Shuttle</Box>
+              {shuttlecalled ? '' : <Input
```

Comment:
syntax probably not correct, but something like this. don't use a ternary for this
```suggestion
              {shuttlecalled && (<Input
```